src/form/state/FormFieldDefinition.test.ts

```diff
@@ -0,0 +1,119 @@
+import { FormFieldDefinition } from './FormFieldDefinition.model';
+
+describe('form field description', () => {
+  describe('display and interaction with value', () => {
+    it.todo('can define the value may be treated as case insensitive');
+
+    it.todo('can define the value must be treated as case sensitive');
+
+    it('can define the value can contain multiple lines of text', () => {
```

Comment:
`can define the value can contain` => `can define the value may contain` might read easier without losing it's meaning?